artifacts/AdditionalSetup.ps1

```diff
@@ -427,7 +427,11 @@ if (($env:cosmoServiceRestart -eq $false) -and ![string]::IsNullOrEmpty($env:saa
 
     Write-Host " - Create user in new tenant (if not exists)"
     if(!(Get-NAVServerUser -ServerInstance $ServerInstance -Tenant $tenantId | Where-Object { $_.UserName.ToLower() -eq $env:username.ToLower() })) {
-        New-NAVServerUser -ServerInstance $ServerInstance -Tenant $tenantId -UserName $env:username -Password $securePassword -AuthenticationEMail $env:username -ErrorAction Continue
+        if ($(env:username).indexOf("@") gt 0) {
```

Comment:
@tfenster sorry I was too fast, I think this should be `-gt`

Reply:
Yep, this is breaking things. I´ll provide a fix PR